test/algorithms/test_min_eigen_optimizer.py

```diff
@@ -353,6 +360,48 @@ def test_samples_vqe(self, simulator):
         self.assertAlmostEqual(results.raw_samples[0].fval, opt_sol)
         self.assertEqual(results.raw_samples[0].status, success)
 
+    def test_vqe_program(self):
```

Comment:
If you want you could merge these two into one test to avoid code duplication like
```python
@data('vqe', 'qaoa')
def test_runtime(self, subroutine):
    if subroutine == 'vqe':
         solver = VQEProgram(...)
    else: 
        solver = QAOAProgram(...)
    
    opt = MinimumEigenOptimizer(solver)
    ...

Reply:
Done in e9289b58a9ad96b074fe7ab3728ef10da0f05d1c.